gemrb/includes/voodooconst.h

```diff
@@ -60,13 +60,6 @@ static const int VOODOO_FINDTRAP_RANGE = 10;
 // The distance of operating a trigger, container, dialog buffer etc.
 static unsigned int MAX_OPERATING_DISTANCE IGNORE_UNUSED = 40; //a search square is 16x12 (diagonal of 20), so that's about two
 
-// used for the shout action, supposedly "slightly larger than the default visual radius of NPCs"
-// while it looks too big, it is needed this big in at least pst (help())
-// bgee testing shows a radius of 72 feet
-// In the EE Help() has the range of "visualrange" * 3 * (16 * (1 or 3/4 in the vertical)) == 3x visual range, converted via Feet2pixels
-// the '48' (3*16) default can be set by the 'Audible Range' game option in baldur.lua
```

Comment:
Please rather edit and move this documentation to the function, so it's clear there was research done and that we shouldn't be changing it without a very good reason. Something like:
```
// audible range was confirmed to be 3x visual range in EEs, accounting for isometric scaling and more than 1x visual rangein others
// in EE, the '48' (3*16) default can be set by the 'Audible Range' game option in baldur.lua
```

The tricky thing is that this is clearly not consistent. The game used a double value of visual range in several places, so our value in the code should actually be 3 * stat / 2.